content/en/docs/reference/scheduling/config.md

```diff
@@ -199,6 +202,9 @@ that are not enabled by default:
   configured labels. This plugin also favors spreading the Pods belonging to a
   Service across nodes.
   Extension points: `PreFilter`, `Filter`, `Score`.
+- `NodePreferAvoidPods`: Prioritizes nodes according to the node annotation
+  `scheduler.alpha.kubernetes.io/preferAvoidPods`.
```

Comment:
This annotation should also be listed in https://kubernetes.io/docs/reference/labels-annotations-taints/

Reply:
Done